RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -5958,3 +5958,69 @@ int meatReserve()
 	
 	return reserve_gnasir + reserve_diary + reserve_island + reserve_extra;
 }
+
+// This function returns the number of bander runaways that will be available after maximizing for fam weight
+int auto_banderRunsAvailable() {
+	if (in_quantumTerrarium()) {
+		if ((my_familiar() == $familiar[Frumious Bandersnatch] && have_skill($skill[The Ode to Booze])) || my_familiar() == $familiar[Pair of Stomping Boots]) {
+			maximize("familiar weight, -tie", false); // Need to maximize or simulate maximizing in accordance with standard autoscend procedure
```

Comment:
You'll want to write a `provideFamiliarWeight()` function for this (probably `provideFamiliarWeight($familiar)` so it can optimise for a specific familiar I guess too). See the auto_providers.ash file for examples.

We don't use naked maximize() calls unless there's absolutely no other choice otherwise as it will end up it'll be flipping equipment back and forth all over the place causing tons of server hits.

Reply:
In the midst of trying to put together a functioning, helpful `provideFamiliarWeight`, thanks for the tip!